MoleculeVAE-reImplementation/train.py

```diff
@@ -167,30 +169,33 @@ def load_data_by_task(params):
             -test_dataset,
         '''
     if params['do_prop_pred']:
-        X_train, X_test, Y_train, Y_test, character_index_lookup_dict = load_dataset(params)
+        X_train, X_test, Y_train, Y_test, _ = load_dataset(params)
     else:
-        X_train, X_test, character_index_lookup_dict = load_dataset(params)
+        X_train, X_test, _ = load_dataset(params)
 
     train_dataset = CustomMoleculeDataset(X_train, Y_train)
-    train_set_final, valid_set_final = split_validation_dataset(train_dataset, percentage_train=0.8)
+    train_dataset, valid_dataset = split_validation_dataset(
+        train_dataset, percentage_train=0.8)
     test_dataset = CustomMoleculeDataset(X_test, Y_test)
-    test_set_final, _ = split_validation_dataset(test_dataset, percentage_train=1)# To control the number of percentage
-    training_data_dic = pd.DataFrame(list(zip( train_set_final, valid_set_final, test_dataset )),
```

Comment:
This seemed to be a bit confused. The variable is called `..._dic` but you're making it into a `DataFrame`, and the items going into the `DataFrame` are 2 `torch.DataLoader` objects and one `torch.DataSet` object. This caused my VM to crash, presumably because it wasn't sure what to do with that output.
Later `DataFrame` like actions (like `to_list` get called on this output but in the final `train` and `test` calls they seem to be being treated as `DataLoader` or `DataSet` types. 

I tried to straighten it all out by making `training_data_dic` and dictionary holding `DataSet` objects and did the conversion to `DataLoaders` inside the `train` and `test` functions.